ibm_was/datadog_checks/ibm_was/ibm_was.py

```diff
@@ -76,7 +76,7 @@ def get_node_from_name(self, xml_data, path):
             return []
 
     def get_node_from_root(self, xml_data, path):
-        return xml_data.xpath('//{}'.format(path))
+        return xml_data.findall(path)
```

Comment:
Not really a need for xpath expression here

Reply:
```suggestion
        return xml_data.xpath('.//{}'.format(path))
```

Seems the issue with `get_node_from_root` is the same as for `get_node_from_name`. 

I would suggest a similar approach for 1) consistency and 2) is match better the current naming of the function and parameters. Both methods work though.

Reply:
Yeah, I thought that having that xpath was unnecessary complexity